bazel/repository_locations.bzl

```diff
@@ -302,4 +302,9 @@ REPOSITORY_LOCATIONS = dict(
         sha256 = "9ca76ac1c22d9360936006efddf992977ebf8e4788ded8e5f9d511285c9ac774",
         urls = ["https://github.com/fuzzitdev/fuzzit/releases/download/v2.4.76/fuzzit_Linux_x86_64.zip"],
     ),
+    upb = dict(
+        sha256 = "61d0417abd60e65ed589c9deee7c124fe76a4106831f6ad39464e1525cef1454",
+        strip_prefix = "upb-9effcbcb27f0a665f9f345030188c0b291e32482",
+        urls = ["https://github.com/protocolbuffers/upb/archive/9effcbcb27f0a665f9f345030188c0b291e32482.tar.gz"],
```

Comment:
@lizan the commit you used here is in the https://github.com/bazelbuild/rules_apple/releases/tag/0.19.0 release of `rules_apple`. Any reason to use the commit vs. the release?